### PR TITLE
Support CT zone with field or subfield

### PR DESCRIPTION
**Pull request by commandgjj**
libOpenflow supports immediate and field/subfield for conntrack zone field. But
ofnet only support zone with immediate. This patch will add zone with
field/subfield support.

Signed-off-by: Jinjun Gao <gjinjun@gmail.com>

### CODE REVIEW

**Comment by commandgjj** [comment]
Hello, guys. Can you help review this patch? Thanks!

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-v0.3.0
+v0.3.1
```
Comment (gran-vmv) [inline VERSION:1]: This should be updated.

File: ofctrl/ofAction.go [hunk 2]
```diff
@@ -518,16 +522,36 @@ func (a *NXConnTrackAction) GetActionType() string {
...
 		force:   force,
 		table:   table,
-		zone:    zone,
+		zoneImm: zone,
 		actions: actions,
 	}
 }
 
+// This function support immediate number and field or subfield for ct_zone
+func NewNXConnTrackActionWithZoneField(commit bool, force bool, table *uint8, zoneImm *uint16, zoneSrcFieldName string, zoneSrcRange *openflow13.NXRange, actions ...openflow13.Action) *NXConnTrackAction {
```
Comment (gran-vmv) [inline ofctrl/ofAction.go:537]: Could you add some testcase like PR #23?

**Comment by gran-vmv** [comment]
@wenyingd Could you review this PR?

**Review by wenyingd — APPROVED** [review]
LGTM

**Comment by commandgjj** [comment]
> LGTM

@wenyingd Can you please help merge this PR? Thanks!

**Pull request merged**